### DISCUSSION TITLE
Detecting errors caused by reaching the limit set in `pest::set_call_limit`?

### DISCUSSION THREAD

**Discussion by rscarson** [pest-parser/pest#977] [Q&A]
When an expression reaches the call limit it just appears to be a syntax error, as if the input was simply truncated
How can I determine if the call limit caused an error in order to provide usable feedback to the user?

**Comment by tomtau** [comment]
I think one can match on the returned error variant to check if it's `ErrorVariant::CustomError` and the error message is "call limit reached"?
